issues/usePatientInfo.ts

```diff
@@ -26,6 +57,7 @@ interface LoggingService {
 export default function usePatientInfo(
   userId: string
 ): PatientInfo | undefined {
+  const isMounted = useMountedState();
   const userService = ServiceFactory.createService<UserService>('userService');
```

Comment:
word "create" as part of `createService` here might indicate that a new instance is created on each invocation, which would then require using `useMemo` or other memoization means.